<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CountdownTimer</class>
 <widget class="QDockWidget" name="CountdownTimer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Countdown Timer</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>469</width>
      <height>449</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="timerDisplayLayout">
       <item>
        <widget class="QLCDNumber" name="timerDisplay">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
         <property name="digitCount">
          <number>8</number>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Filled</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="timerLayout">
       <item>
        <widget class="QLineEdit" name="timerHours">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="timerHoursLabel">
         <property name="text">
          <string>h</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="timerMinutes">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="timerMinutesLabel">
         <property name="text">
          <string>m</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="timerSeconds">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="timerSecondsLabel">
         <property name="text">
          <string>s</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="sourceDropDownLayout" stretch="0,1">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Text Source</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="endMessageLayout" stretch="0,0,1">
       <item>
        <widget class="QCheckBox" name="endMessageCheckBox">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="timerEndLabel">
         <property name="text">
          <string>End Message</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_4"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="sceneDropDownLayout">
       <item>
        <widget class="QCheckBox" name="switchSceneCheckBox">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sceneSwitchLabel">
         <property name="text">
          <string>Switch Scene</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sceneSourceDropDownList"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QPushButton" name="resetButton">
         <property name="text">
          <string>Set/Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pauseButton">
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="playButton">
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
